src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocvariable/InputJavadocVariableInner.java

```diff
@@ -5,29 +5,34 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocvariable;
 
 /**
+ * Config:
+ * scope = private
+ * excludeScope =null
+ *
+ *
  * Tests having inner types
  * @author Oliver Burn
- **/
+ */
 class InputJavadocVariableInner
 {
     // Ignore - two violations
     class InnerInner2
     {
         // Ignore
-        public int fData;
+        public int fData;   // violation
```

Comment:
Keep only a single space between `;` and `//` as it has been followed in rest of the PR.

Reply:
Resolved.